src/components/MapView/Layers/AnalysisLayer/index.tsx

```diff
@@ -46,21 +47,26 @@ function AnalysisLayer({ before }: { before?: string }) {
     };
   }
 
+  const baselineIsBoundary =
```

Comment:
Unfortunately not here, or not easily because this function handles different types of analysis results and BaselineLayerResult is only one of the types.

Or maybe there is a cleaner way to make TypeScript not complain about `boundaryId` not existing on AnalysisResult?

Reply:
Thanks a lot Ryan for these useful right up. I ended up adding the `'baselineLayerId' in analysisData` instead of the type assertion